packages/terra-badge/tests/jest/__snapshots__/Badge.test.jsx.snap

```diff
@@ -1,20 +1,20 @@
 exports[`test should render a badge component in the order, icon and text with medium size 1`] = `
 <span
-  className="terra-Badge terra-Badge--medium terra-Badge--default">
+  className="badge medium default">
   <img
     alt="Test icon" />
   <span
-    className="terra-Badge-text">
+    className="text">
```

Comment:
> Is this something the CSS Modules does?

I believe so, [this webpack config](https://github.com/cerner/terra-core/blob/4edb57182de142580595cd60f61d159cd140b751/packages/terra-site/webpack.config.js#L38) is used to format the generated class names.

This means the nightwatch tests won't be able to rely on specific classNames. We talked about using IDs in nightwatch test files to identify components and then switch to inspecting the actually CSS using http://nightwatchjs.org/api#expect-css or http://nightwatchjs.org/api#assert-cssProperty

The jest tests have access to the CSS classes before they run through css modules / css loader so we can reference class names like "text" or "primary" in the jest tests.